### PR TITLE
[WIP] FUSETOOLS2-252: Create UI Tests for Usecase #1 - Create the VDB

### PR DESCRIPTION
**Pull request by djelinek**

### CODE REVIEW

**Comment by djelinek** [comment]
Hi @lhein @apupier 

Still work in progress.. but first UI Tests are coming..

I need to replace some remaining static waitings with dynamic wait conditions and maybe do some refactoring, but I would be happy for feedback what I could missed for first VDB usecase. Thanks!

File: src/ui-test/create-vdb.smoketest.ts [hunk 1]
```diff
@@ -0,0 +1,165 @@
+import { expect } from 'chai';
```
Comment (apupier) [inline src/ui-test/create-vdb.smoketest.ts]: missing header

File: src/ui-test/datavirt-extension-info.smoketest.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import { EditorView, ExtensionsViewItem } from 'vscode-extension-tester';
```
Comment (apupier) [inline src/ui-test/datavirt-extension-info.smoketest.ts]: missing header

**Review by apupier — COMMENTED** [review]
- if I understand well, most of the tests depends on the previous one. Are you sure that Mocha tests always runs in the declared order? (not found reliable answers with 2 minutes search)
- scenario tested is good

**Comment by djelinek** [comment]
> * if I understand well, most of the tests depends on the previous one. Are you sure that Mocha tests always runs in the declared order? (not found reliable answers with 2 minutes search)

Hi @apupier,  I also did not found documented notes about order of tests but from my experiences it was always executed as it is declared. Only thing I noticed is order of whole suites (each .js file) because here is used alphabetical order.

But as I said I am working now on waiting conditions which are now missing and I will also try to do some modifications into whole describe suite to be more with Mocha conventions

**Comment by lhein** [comment]
remember to upgrade extension tester to 3.0.0+

**Comment by djelinek** [comment]
DataVirt has been decomissioned, so work on the extension has been stopped too.

**Pull request closed without merging**